README.md

```diff
@@ -47,24 +47,28 @@ Usage:
-      --geth.syncmode string      Geth blockchain sync mode (fast, full, light)
-      --geth.gcmode string        Geth garbage collection mode (full, archive)
-      --geth.cache string         Memory (in MB) allocated to geth's internal caching
-      --geth.maxpeers string      Maximum number of network peers (network disabled if set to 0) (default: 1100)
-      --geth.ipcpath string       Path to the geth ipc file
-      --geth.logfile string       Path to logs file
-      --geth.publicip string      Public Ip to configure geth (sometimes required for discovery)
-      --geth.staticnodes string   StaticNodes to use (separated by ,)
-      --geth.verbosity string     Geth log verbosity (number between [1-5])
-  -h, --help                      help for run
-      --rpc.address string        Listening address for http server
-      --rpc.port uint             Listening port for http server (default 8080)
-      --rpc.reqTimeout duration   Timeout when serving a request (default 25s)
+      --datadir string             datadir to use
+      --geth.binary string         Path to the celo-blockchain binary
+      --geth.bootnodes string      Bootnodes to use (separated by ,)
+      --geth.cache string          Memory (in MB) allocated to geth's internal caching
+      --geth.gcmode string         Geth garbage collection mode (full, archive) (default "full")
+      --geth.genesis string        (Optional) path to the genesis.json, for use with custom chains
+      --geth.ipcpath string        Path to the geth ipc file
+      --geth.logfile string        Path to logs file
+      --geth.maxpeers string       Maximum number of network peers (network disabled if set to 0) (default: 1100) (default "1100")
+      --geth.network string        Network to use, either 'mainnet', 'alfajores', or 'baklava'
+      --geth.publicip string       Public Ip to configure geth (sometimes required for discovery)
+      --geth.rpcaddr string        Geth HTTP-RPC server listening interface (default "127.0.0.1")
+      --geth.rpcport string        Geth HTTP-RPC server listening port (default "8545")
+      --geth.rpcvhosts string      Geth comma separated list of virtual hostnames from which to accept requests (default "localhost")
+      --geth.staticnodes string    StaticNode to use (separated by ,)
+      --geth.syncmode string       Geth blockchain sync mode (fast, full, light) (default "fast")
+      --geth.verbosity string      Geth log verbosity (number between [1-5])
+  -h, --help                       help for run
+      --rpc.address string         Listening address for http server
+      --rpc.port uint              Listening port for http server (default 8080)
+      --rpc.reqTimeout duration    Timeout for requests to this service, this also controls the timeout sent to the blockchain node for trace transaction requests (default 25s)
+      --txtrace.Timeout duration   Timeout to use when when making trace transaction requests to a full node (default 50s)
```

Comment:
Is this arg used? I thought that `rpc.reqTimeout` is now used for both? (That said, I think having two separate args if possible may be useful IMO)

Reply:
No that is a phantom arg! I started with a separate flag, but given that the `rpc.ReqTimeout` would override the timeout set for tracing I thought it was simpler just to have one. And then I promptly forgot to update the docs! thanks!